theme/highlight.js

```diff
@@ -0,0 +1,5588 @@
+/*
```

Comment:
Is this the same file included again?

Reply:
yeah, i put the comment in the wrong file. there will be 2 copies of the highlighter file, one in `/javascript` which is the source and one in `/book/javascript` which is generated by `mdbook build`. Happy to move the file somewhere else if we don't want it in `/javascript`